docs/porting/overview.md

```diff
@@ -1,5 +1,82 @@
-# 🔧 Overview of porting process
+# Overview of the Porting Process
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
\ No newline at end of file
+By [Phillip Carter](https://github.com/cartermp)
+
+If you've got code running on .NET Framework you may be interested in running your code on .NET Core.  This article covers evaluating if porting to .NET Core is reasonable for you, an overview of the porting process, and a list of the tools you may use to port.
+
+## Should you port?
+
+The first thing you need to do is evaluate if it makes sense to port .NET Framework code over to .NET Core in the first place.  .NET Core offers some attractive capabilities, such as:
+
+- Using ASP.NET Core to write great web apps and services which you can write and deploy on OS X, supported Linux distributions, or Windows
+- The ability to deploy apps and services with different versions of the .NET Core runtime installed side-by-side
+- The ability to write a UWP app that will run on multiple Windows 10 devices
+- The ability to utilize the advancements of .NET Core while your code lives seamlessly alongside .NET Framework code, such as a WinForms GUI consuming .NET Core microservices
+
+Your code could be a good candidate for porting to .NET Core if...
+
+- You have an ASP.NET web app that you'd like to deploy to more than .NET Framework
+- Your code is a library and you want to expand its reach
+- You have a compelling business case to run code on more than just Windows
+- You want to take advantage of more flexible deployment
+- You have expertise on non-Windows platforms you wish to leverage
+- You're looking for an opportunity to implement an architectural change for your services (e.g. a monolithic service to microservices deployed to containers)
+- You wish to take advantage of performance advancements made in .NET Core and ASP.NET Core
+- You love new open source technology and want to cut your teeth on .NET Core
+
+Your code may not be a good candidate for porting if...
+
+- Your code is tied to a Windows GUI or depends on Windows-specific features
+- You have no business needs or expertise which would warrant running on more than Windows
+- You don't have a need for more flexible deployment or self-contained console apps
+- You're happy with your code running on .NET Framework as it is today
+- Your code depends heavily on COM interop
+
+If it makes sense to port now, then read on!
```

Comment:
why now? Also read on sounds too informal to me, more appropriate for a blog post or something. Not sure what others think. Maybe something like "If it makes sense to port your code, then continue to read."